app/Notifications/Mship/SlackInvitation.php

```diff
@@ -30,7 +29,7 @@ public function __construct()
      */
     public function via($notifiable)
     {
-        if (Gate::forUser($notifiable)->allows('register-slack')) {
+        if (!$notifiable->slack_id) {
```

Comment:
```suggestion
         return !$notifiable->slack_id ? ['mail', 'database'] : [];
```
Not a big one but might make it a bit simpler? 6 of one, half a dozen of the other on this one.

Reply:
If I look at your code, I recognise it as "proper".
But when I glance at both, I instantly know what mine is doing. There is a degree of "mental debt" being added by decoding the `!` then the `?` etc with your suggestion.

Maybe I'm just not very good! 😉

Reply:
I hear ya! To me, its good to look at because I know left is true and right is false. I'm not going to block it based on this though :laughing:

Reply:
@AxonC Good, get it approved 💯